osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -625,6 +631,30 @@ public SkinEditorToast(LocalisableString value, string skinDisplayName)
             }
         }
 
+        public partial class ResetConfirmDialog : PopupDialog
```

Comment:
Should arguably reuse `DeleteConfirmationDialog`? Naming is unfortunate there, but you could retrofit it into a `DangerousActionDialog` or something?

Reply:
Tangentially related I didn't see it at a glance but I find the use of "dialog" instead of "dialogue" a tad jarring given the use of UK / AU English in most places and it being seen as the proper way to do things within the project. Not that important, and probably not worth the 500 file diffstat fixing it would incur, just weird